<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Acquaintance</class>
 <widget class="QWidget" name="Acquaintance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="charmanderContainer" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>240</width>
     <height>420</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#charmanderContainer {
	border : 5px solid white;
	border-radius : 10px;	
	background-color: rgba(255, 100, 0, 75);
}</string>
   </property>
   <widget class="QLabel" name="charmanderIntroduction">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>A fire burns at the tip of this Pokémon's slender tail, and has blazed there since Charmander's birth. The flame can be used as an indication of Charmander's health and mood, burning brightly when the Pokémon is strong, weakly when exhausted.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="charmanderPhoto">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>200</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/charmander);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="charmanderButton">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>365</y>
      <width>230</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Verdana&quot;;
color : white;
border : 2px solid rgba(0, 0, 0, 25);
border-bottom-left-radius : 5px;
border-bottom-right-radius : 5px;
background-color : rgba(0, 0, 0, 75);</string>
    </property>
    <property name="text">
     <string>I WANT YOU</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>charmander</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="squirtleContainer" native="true">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>30</y>
     <width>240</width>
     <height>420</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#squirtleContainer {
	border : 5px solid white;
	border-radius : 10px;
	background-color: rgba(0, 100, 255, 75);
}</string>
   </property>
   <widget class="QLabel" name="squirtleIntroduction">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>Its body is encased by a tough shell that forms and hardens after birth. Squirtle's shell is a useful tool. It can withdraw into the shell for protection or for sleeping, and its grooved, rounded shape helps to reduce water resistance when this Pokémon swims.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="squirtlePhoto">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>200</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/squirtle);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="squirtleButton">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>365</y>
      <width>230</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Verdana&quot;;
color : white;
border : 2px solid rgba(0, 0, 0, 25);
border-bottom-left-radius : 5px;
border-bottom-right-radius : 5px;
background-color : rgba(0, 0, 0, 75);</string>
    </property>
    <property name="text">
     <string>I WANT YOU</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>squirtle</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="bulbasaurContainer" native="true">
   <property name="geometry">
    <rect>
     <x>562</x>
     <y>30</y>
     <width>240</width>
     <height>420</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#bulbasaurContainer {
	border : 5px solid white;
	border-radius : 10px;
	background-color: rgba(0, 255, 100, 75);
}</string>
   </property>
   <widget class="QLabel" name="bulbasaurIntroduction">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>Bulbasaur has shown itself to be an excellent caretaker, even having a special technique called the &quot;Bulba-by.&quot; To perform this technique, Bulbasaur uses its vines to pick up a young Pokémon and soothingly rocks it in the air.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="bulbasaurPhoto">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>200</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/bulbasaur);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bulbasaurButton">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>365</y>
      <width>230</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Verdana&quot;;
color : white;
border : 2px solid rgba(0, 0, 0, 25);
border-bottom-left-radius : 5px;
border-bottom-right-radius : 5px;
background-color : rgba(0, 0, 0, 75);</string>
    </property>
    <property name="text">
     <string>I WANT YOU</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>bulbasaur</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
